### PR TITLE
[packages] add ivy 1.3.1

### PR DESCRIPTION
**Pull request by flixr**
New 1.3.1 version of the OCaml bindings for the Ivy bus C library.

### CODE REVIEW

File: packages/ivy/ivy.1.3.1/url [hunk 1]
```diff
@@ -0,0 +1 @@
+archive: "http://www.eei.cena.fr/products/ivy/download/packages/ivy-ocaml_1.3.1.tar.xz"
```
Comment (Chris00) [inline packages/ivy/ivy.1.3.1/url]: Missing checksum

File: packages/ivy/ivy.1.3.1/opam [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+]
+install: [make "install"]
+remove: [
+  ["ocamlfind" "remove" "ivy"]
+  ["ocamlfind" "remove" "glibivy"]
+  ["ocamlfind" "remove" "tkivy"]
+]
+depends: "ocamlfind" {build}
+depexts: [
+  [["debian"] ["libpcre3-dev"]]
```
Comment (Chris00) [inline packages/ivy/ivy.1.3.1/opam]: `[["debian"] ["libpcre3-dev" "tk-dev" "libglib2.0-dev"]]` etc.

Comment (flixr) [inline packages/ivy/ivy.1.3.1/opam]: Ok, will change that... just copied it from the older ivy version where someone else put it in like that.

File: packages/ivy/ivy.1.3.1/opam [hunk 3]
```diff
@@ -0,0 +1,23 @@
+opam-version: "1.2"
+maintainer: "gautier.hattenberger@enac.fr"
+version: "1.3.1"
+homepage: "http://www.eei.cena.fr/products/ivy"
+build: [
+  [make]
```
Comment (Chris00) [inline packages/ivy/ivy.1.3.1/opam]: Use `"--prefix" prefix` to select the proper location (maybe for the install step).

**Comment by flixr** [comment]
Ok, added checksum and updated depexts.

What is actually tested on travis? I mean this package just provides OCaml bindings for the ivy-c lib.
The script in the "source linux" entry would download and install this dependency (but require a debian based system.
Or what is the preferred way to handle such a case?

File: packages/ivy/ivy.1.3.1/opam (hunk 3 above)
Comment (flixr) [inline packages/ivy/ivy.1.3.1/opam]: There is no prefix in the Makefile for ivy-ocaml... it uses ocamlfind to install the libs...

Comment (Chris00) [inline packages/ivy/ivy.1.3.1/opam]: On Tue, 20 Jan 2015 07:02:51 -0800, Felix Ruess wrote:
> 
> There is no prefix in the Makefile for ivy-ocaml... it uses ocamlfind to
> install the libs...

OK

**Comment by flixr** [comment]
I've updated the gist to pull the latest ivy-c tarball, which should build now... so maybe you can re-trigger the build?
The source install script is https://gist.github.com/flixr/10003993
Should I keep the apt-get install lines in there? They seem to be handled by the debian/ubuntu depexts already...

**Comment by avsm** [comment]
Travis restarted.  The source fragments runs after depext, but there's no harm in repeating them...

**Comment by flixr** [comment]
Is there a way to invoke the source script as a normal user who wants to install this via opam?
Then I would probably just remove the apt-get line as it will only work on debian based systems...

**Comment by flixr** [comment]
Seems the re-run build passed: https://travis-ci.org/ocaml/opam-repository/builds/47654969
But it was re-run on the first commit (and not the second one with the checksum) so it is still marked as failed...

**Comment by samoht** [comment]
I've re-run the tests for the second commit. The magic runes to install the "sources" for a user is to do:
```
opam install opam-installext
opam-installext PACKAGE # opam installext (without `-`) should work as well
```

**Comment by flixr** [comment]
Thanks @samoht !

**Pull request merged**